backend/dataall/tasks/data_sharing/share_managers/s3_share_manager.py

```diff
@@ -410,12 +411,8 @@ def handle_share_failure(self, error: Exception) -> bool:
             f'with target account {self.target_environment.AwsAccountId}/{self.target_environment.region} '
             f'due to: {error}'
         )
-        AlarmService().trigger_folder_sharing_failure_alarm(
-            self.target_folder, self.share, self.target_environment
-        )
-        return True
 
-    def handle_revoke_failure(self, error: Exception) -> bool:
+    def handle_revoke_failure(self, error: Exception) -> None:
```

Comment:
Please update the function description, maybe also consider renaming the function as it only logs now.